src/utils/getFormattedDate.js

```diff
@@ -0,0 +1,9 @@
+export default function getFormattedTime(timestamp, options) {
+  const { format: formatTime } = new Intl.DateTimeFormat('en', {
+    hour: options.hourOptions,
+    minute: options.minuteOptions,
+    hour12: false,
+  })
+
+  return formatTime(timestamp)
+}
```

Comment:
Use named exports for utils. Also, pass all options to `DateTimeFormat`, you may need it in the future.

```suggestion
export const timeFormatter = (options) =>
  new Intl.DateTimeFormat('en', {
    hour: '2-digit',
    minute: '2-digit',
    hour12: false,
    ...options,
  })
```

Reply:
Arrow functions are ok, I forgot to use it this time. By the way, thank you for your suggestion